.github/workflows/core_build.yml

```diff
@@ -0,0 +1,59 @@
+name: core-build
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        compiler: [clang6, clang7]
+        modules: [with, without]
+    runs-on: ubuntu-18.04
+    name: ${{ matrix.compiler }}-${{ matrix.modules }}-modules
+    env:
+      COMPILER: ${{ matrix.compiler }}
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
what does `actions/checkout@v2` mean?

Reply:
Its the standard way to do git clone in github actions.
Travis does that step automatically before your script starts to run.
If you create a new github action with default template it uses that command as the first step to download your repository code: https://i.imgur.com/zTbtGzl.png